client/components/jetpack/jetpack-product-info/hooks/use-included-product-description-map.tsx

```diff
@@ -128,7 +129,11 @@ export const useIncludedProductDescriptionMap = ( productSlug: string ) => {
 				productSlug
 			);
 
-			if ( isJetpackCompletePlan ) {
+			const isJetpackGrowthPlan = ( JETPACK_GROWTH_PLANS as ReadonlyArray< string > ).includes(
```

Comment:
Without this the lightbox content breaks.

Reply:
Instead of having this here, could we add it to the Calypso Products package like we have here: fbhepr%2Skers%2Spnylcfb%2Scnpxntrf%2Spnylcfb%2Qcebqhpgf%2Sfep%2Svf%2Qwrgcnpx%2Qpbzcyrgr%2Qfyht.gf%3Se%3Q3r87r7rn-og?

I see that we are doing the same thing with Complete here, but if it's possible I think it would be great just to use those package functions instead of checking manually here

Reply:
I fixed the link, the question mark was messing it up. However here is the public facing link, probably better anyway: https://github.com/Automattic/wp-calypso/blob/trunk/packages/calypso-products/src/is-jetpack-complete-slug.ts

Reply:
Thanks for the suggestion, it allowed me to unify [this](https://github.com/Automattic/wp-calypso/pull/96575/commits/341187ff68cd16055de2c53c546cae08318bf9ce#diff-c8b7cf7157b2880abc8f1ec487b4b86bd5f2cd4b59540e88c62a8c4dfd1d7370L6) place as well.